src/Http/Routing/src/RoutingMetrics.cs

```diff
@@ -10,33 +10,30 @@ internal sealed class RoutingMetrics
     public const string MeterName = "Microsoft.AspNetCore.Routing";
 
     private readonly Meter _meter;
-    private readonly Counter<long> _matchSuccessCounter;
-    private readonly Counter<long> _matchFailureCounter;
+    private readonly Counter<long> _matchAttemptsCounter;
 
     public RoutingMetrics(IMeterFactory meterFactory)
     {
         _meter = meterFactory.Create(MeterName);
 
-        _matchSuccessCounter = _meter.CreateCounter<long>(
-           "routing-match-success",
-            description: "Number of requests that successfully matched to an endpoint.");
-
-        _matchFailureCounter = _meter.CreateCounter<long>(
-           "routing-match-failure",
-            description: "Number of requests that failed to match to an endpoint. An unmatched request may be handled by later middleware, such as the static files or authentication middleware.");
+        _matchAttemptsCounter = _meter.CreateCounter<long>(
+           "aspnet.routing.match_attempts",
+            description: "Number of requests that were attempted to be matched to an endpoint.");
     }
 
-    public bool MatchSuccessCounterEnabled => _matchSuccessCounter.Enabled;
+    public bool MatchSuccessCounterEnabled => _matchAttemptsCounter.Enabled;
 
     public void MatchSuccess(string route, bool isFallback)
     {
-        _matchSuccessCounter.Add(1,
-            new KeyValuePair<string, object?>("route", route),
-            new KeyValuePair<string, object?>("fallback", isFallback));
+        _matchAttemptsCounter.Add(1,
+            new KeyValuePair<string, object?>("http.route", route),
+            new KeyValuePair<string, object?>("aspnet.routing.match_status", "success"),
```

Comment:
btw, maybe we don't need this attribute - the presence of a route indicates a successful match already.

https://github.com/lmolkova/semantic-conventions/pull/1#discussion_r1279851076

Reply:
Ok. Will remove it.